src/components/Button.js

```diff
@@ -24,4 +25,16 @@ const Button = ({ children, className = '', size }) => {
   );
 };
 
+Button.propTypes = {
+  children: PropTypes.node,
+  className: PropTypes.string,
+  size: PropTypes.string,
+};
+
+Button.defaultProps = {
+  children: null,
+  className: '',
+  size: '',
```

Comment:
Should we default to a large sized button?

Reply:
Good idea, I will make that change next time we work on the Button component.